lib/IDE/CodeCompletion.cpp

```diff
@@ -1004,6 +1007,9 @@ void CodeCompletionResultBuilder::addCallArgument(
     addChunkWithText(ChunkKind::CallArgumentType, TypeName);
   }
 
+  if (HasDefault)
+    addSimpleChunk(ChunkKind::CallArgumentDefault);
```

Comment:
If we want to add the actual argument value to the type, we probably want it like `<callarg.default>=  nil</callarg.default>` Does it make sense to make this like
```cpp
withNestedGroup(ChunkKind::CallArgumentDefaultBegin, []() {
  // TODO?: Add the actual value?
})
```

Reply:
Sure, seems reasonable.